mermithid/processors/TritiumSpectrum/FakeDataGenerator.py

```diff
@@ -125,9 +129,39 @@ def InternalConfigure(self, params):
                                                         self.survival_prob,
                                                         self.NScatters)
             elif self.lineshape=='detailed':
-                if not os.path.exists('./scatter_spectra_files'):
-                    raise IOError('./scatter_spectra_files does not exist')
+                # check path exists
+                if 'scatter_spectra_file' in self.detailed_scatter_spectra_path:
```

Comment:
Should we retain an option for the processor to raise an error if it doesn't find "scatter_spectra_file" in either of the places it checks?

Reply:
no because now that the fake data generator uses the complex lineshape processor.  the lineshape processor looks for these files in the same path that the oscillator strength files are located in and if these files do not exist they will be created.

Reply:
Great, thanks for the explanation!